cmd/cli/kubectl-kyverno/report/report_test.go

```diff
@@ -268,6 +268,27 @@ func TestComputePolicyReportResult(t *testing.T) {
 			Category:  "Pod Security Standards (Restricted)",
 			Severity:  policyreportv1alpha2.SeverityMedium,
 		},
+	}, {
+		name:           "fail",
+		auditWarn:      false,
+		engineResponse: engineapi.NewEngineResponse(unstructured.Unstructured{}, engineapi.NewKyvernoPolicy(policy), nil),
+		ruleResponse:   *engineapi.RuleWarn("xxx", engineapi.Mutation, "test"),
+		want: policyreportv1alpha2.PolicyReportResult{
+			Source: "kyverno",
+			Policy: "psa",
+			Properties: map[string]string{
+				"controls": "allowPrivilegeEscalation, capabilities_restricted, runAsNonRoot, seccompProfile_restricted",
+				"standard": "restricted",
+				"version":  "xxx",
+			},
+			Rule:      "xxx",
+			Result:    policyreportv1alpha2.StatusError,
+			Resources: []corev1.ObjectReference{{}},
+			Message:   "test",
+			Scored:    true,
+			Category:  "Pod Security Standards (Restricted)",
+			Severity:  policyreportv1alpha2.SeverityMedium,
+		},
```

Comment:
I have added this test, but even if I remove it, still the tests are failing. I checked locally to via `make test-unit` command.

Reply:
You can check the errors in the run logs:

https://github.com/kyverno/kyverno/actions/runs/8107195987/job/22158313096?pr=9785#step:5:1888

If you run the test locally in a debugger, you should be able to see why the result does not match the expected result.

Reply:
Yep